### PR TITLE
Build images from scratch

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Build images from scratch
- Fixed HB latest release

**Related issue(s)**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -53,7 +53,7 @@ release: tar-chart append-changelog release-branch
 	./hack/release/push_release.sh $(GIT_TAG) $(GIT_REPO)
 
 .PHONY: latest-release
-latest-release: set-latest-tag tag-chart-images tar-chart append-changelog update-release-docs
+latest-release: set-latest-tag tag-chart-images tar-chart update-release-docs append-changelog
```
Comment (polskikiel) [inline Makefile]: it's a fix for generating the latest release on each commit

File: internal/controller/common.go [hunk 2]
```diff
@@ -181,13 +181,15 @@ func (c *common) OnAdd(addon *internal.CommonAddon, lastStatus v1alpha1.CommonAd
 			}
 		}
 	}
-	if saved || len(deletedAddonsIDs) > 0 {
+	removed := len(deletedAddonsIDs) > 0
```
Comment (polskikiel) [inline internal/controller/common.go]: I've only improved readability here

File: internal/storage/storage.go [hunk 3]
```diff
@@ -93,6 +97,20 @@ func (cl *ConfigList) ExtractEtcdURL() string {
...
 }
 
+// WaitForEtcdReadiness waits for ETCD to be ready
+func (cl *ConfigList) WaitForEtcdReadiness() error {
+	return wait.Poll(time.Second, time.Second*30, func() (bool, error) {
+		resp, err := http.Get(cl.ExtractEtcdURL() + "/health")
+		if err != nil {
+			return false, nil
+		}
+		if resp.StatusCode == http.StatusOK {
```
Comment (mszostok) [inline internal/storage/storage.go]: same here, log this info when timeout occured

File: internal/storage/storage.go [hunk 4]
```diff
@@ -93,6 +97,20 @@ func (cl *ConfigList) ExtractEtcdURL() string {
 	return etcdURL
 }
 
+// WaitForEtcdReadiness waits for ETCD to be ready
+func (cl *ConfigList) WaitForEtcdReadiness() error {
+	return wait.Poll(time.Second, time.Second*30, func() (bool, error) {
+		resp, err := http.Get(cl.ExtractEtcdURL() + "/health")
+		if err != nil {
```
[the remark this answers is not in the document]
Comment (polskikiel) [reply internal/storage/storage.go]: done

**Pull request merged**